### DISCUSSION TITLE
RadzenDropDown filtering  doesn't work.

### DISCUSSION THREAD

**Discussion by jmlpiqueras** [Blazored/Modal#351] [Q&A]
I have a RadzenDropDown control with Filtering mode allowed. But when I click on the control, the input search control (in the RadzenDropDown)  doesn't show the cursor line, and I can't type text. Do you know how to solve that?

**Comment by chrissainty** [comment]
Have you spoken to Radzen as it's their control?

**Reply by jmlpiqueras** [reply]
I think I have not explained well. The RadzenDropDown works very well out of Modal view. If i show the control in a Modal view, its search engine doesn't work. If anyone has any advice i would appreciate it.
Blazored Modal is a great component and it would be great if they could work together (Modal and RadzenDropDown).
